COPY/etc/httpd/conf.d/manageiq-balancer-websocket.conf

```diff
@@ -1,12 +1,15 @@
+# Temporarily disable the connection reuse for WebSockets for httpd version < 2.4.25
+# https://bugzilla.redhat.com/show_bug.cgi?id=1404354
+
 <Proxy balancer://evmcluster_websocket/ lbmethod=bybusyness>
-BalancerMember ws://0.0.0.0:5000
-BalancerMember ws://0.0.0.0:5001
-BalancerMember ws://0.0.0.0:5002
-BalancerMember ws://0.0.0.0:5003
-BalancerMember ws://0.0.0.0:5004
-BalancerMember ws://0.0.0.0:5005
-BalancerMember ws://0.0.0.0:5006
-BalancerMember ws://0.0.0.0:5007
-BalancerMember ws://0.0.0.0:5008
-BalancerMember ws://0.0.0.0:5009
+BalancerMember ws://0.0.0.0:5000 disablereuse=On
```

Comment:
@skateman This is nitpicky but it looks like you can use `on` (lowercase) so it's consistent throughout this 
file: http://blog.moserit.com/tuning-apaches-mod_proxy-and-tomcat-servers

Sorry.  🏃